scoreboard-rules.md

```diff
@@ -0,0 +1,44 @@
+# Quint Scoreboard Rules
+
+We appreciate the efforts of our peers for going through our minimal viable
+solution -- a codeword for buggy tooling and documentation and nobody is sure
+how exactly it should be working -- and generating excellent suggestions for
+improvement. To reflect this, we are collecting four kinds of scores at the
+[scoreboard][] on the main page[^1].
+
+## Learning the language [earth :earth_africa:]
+
+The tutorials contain secret codes that you can submit, in order to get
+:earth_africa: scores. Either submit a secret code in a DM to @konnov (if you
+know how to reach him), or create a secret [gist](https://gist.github.com/),
+write your secret code in the gist and mention @konnov there.
+
+## Github contributions [air :cyclone:]
+
+ - A few-line fix in the documentation: **3** :cyclone:
+
+ - A confirmed bug report for something that has a simple workaround: **5**
+   :cyclone:
+
+ - A confirmed bug report that has no simple workaround: **10**
+   :cyclone:
+
+ - A confirmed bug report that leads to language refactoring: **20** :cyclone:
```

Comment:
I'm not sure what is meant by "language refectory"

Reply:
What I have in mind is an issue that led to a change in the language. For instance, #483